src/Database/models/user.js

```diff
@@ -0,0 +1,25 @@
+import { Model } from 'sequelize';
+import config from '../config/db.config';
+
+
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+
+    static associate(models) {
+      User.hasMany(models.Blog, { foreignKey: 'userId', as: 'Blog'});
```

Comment:
you can change this `as` field to a more comprehensive value like `Blogs` because it has many...

Reply:
sure

Reply:
But I also haven't optimized well my associations since I haven't worked on my user's routes yet